features/openhab-addons-external/src/main/resources/conf/gcal.cfg

```diff
@@ -0,0 +1,10 @@
+# Credentials for Google API and Google Calendar name
+
+# Copied from Credentials page on console.developers.google.com
+#client_id=
+
+# Copied from Credentials page on console.developers.google.com
+#client_secret=
+
+# The name of the Google Calendar that GCal should use
+#calendar_name=
```

Comment:
Looks good, but I worry that users might not know which are required or optional.  So a bit more information as described [here](https://github.com/openhab/openhab/wiki/GCal-Binding#obtain-the-credentials) might avoid head scratching or hair pulling.  Also, when you've updated the helpfulness of the comments a bit, could you also fix openhab_default.cfg to match (adding back the `gcal:` prefixes in that version)?